src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java

```diff
@@ -216,7 +218,6 @@ public void deleteRunningInstanceOfApicurito() {
 
     @Then("clean openshift after operatorhub test")
     public void cleanOpenshiftAfterOperatorhubTest() {
-        String csv = "fuse-apicurito.v" + ReleaseSpecificParameters.APICURITO_CURRENT_VERSION + ".0";
```

Comment:
Change it in method `checkThatNameAndImageOfOperatorInOperatorhubAreCorrect()` as well